buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -177,7 +177,11 @@ class BeamModulePlugin implements Plugin<Project> {
      * By default, skips anything in the `org.apache.beam.model` namespace.
      */
     List<String> generatedClassPatterns = [
-      "^org\\.apache\\.beam\\.model.*"
+      "^org\\.apache\\.beam\\.model\\.pipeline\\.v1\\.*",
```

Comment:
Doesn't `^org\\.apache\\.beam\\.model.*` cover all of the these except the windmill worker?

Reply:
Yes. Those changes weren’t necessary. Wasn’t even necessary to add windmill worker...